packaging/linux/common/khiops

```diff
@@ -1,59 +1,65 @@
 #!/bin/bash
 
 # Run Khiops tool
-# see the khiops-env script for full documentation on the environment variables used by Khiops
-
-
+# see the khiops_env script for full documentation on the environment variables used by Khiops
 
 error() {
```

Comment:
I would factor out `warning()` code into a helper function (perhaps with its name mangled so that it is not easy to use directly), then call this helper function in `warning()`; also call it in `error()`, then add the `exit 1` line.